mongo_connector/connector.py

```diff
@@ -1175,6 +1181,13 @@ def main():
     log_startup_info()
 
     connector = Connector.from_config(conf)
+
+    # register a SIGTERM handler to cleanup the connector gracefully
+    def sigterm_handler(signum, frame):
+        connector.signal = ('SIGTERM', signum)
```

Comment:
It doesn't look like we use `signum` anywhere. It would also be nice to handle SIGINT in the same way as SIGTERM, so we know we're handling cleanup/shutdown in the same way.